mautrix_instagram/user.py

```diff
@@ -353,6 +353,9 @@ async def on_disconnect(self, evt: Disconnect) -> None:
     async def on_proxy_update(self, evt: ProxyUpdate | None = None) -> None:
         if self.client:
             self.client.setup_http(self.state.cookies.jar)
+        # Evt comes from MQTT, so if set we expect it to update itself
```

Comment:
We _could_ always do this but then we have `MQTT.connection -> user.on_proxy_update -> MQTT.setup_proxy` which feels wrong?